keep/api/models/db/migrations/versions/2024-12-02-20-42_c6e5594c99f8.py

```diff
@@ -0,0 +1,77 @@
+"""add first_timestamp field to LastAlert
+
+Revision ID: c6e5594c99f8
+Revises: bdae8684d0b4
+Create Date: 2024-12-02 20:42:33.311541
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import mysql
+from sqlalchemy.orm import Session
+
+# revision identifiers, used by Alembic.
+revision = "c6e5594c99f8"
+down_revision = "bdae8684d0b4"
+branch_labels = None
+depends_on = None
+
+
+def populate_db():
+    session = Session(op.get_bind())
+
+    timestamps = session.execute("""
+        select 
+            alert.fingerprint, 
+            alert.tenant_id, 
+            min(alert.timestamp) as first_timestamp 
+        from alert 
+        group by 
+            alert.fingerprint, 
+            alert.tenant_id 
+    """)
+
+    for fingerprint, tenant_id, first_timestamp in timestamps:
+        session.execute("""
+            update 
+                lastalert
+            set
+                lastalert.first_timestamp = :first_timestamp
+            where
+                lastalert.fingerprint = :fingerprint
+                and lastalert.tenant_id = :tenant_id
+        """, {
+            "first_timestamp": first_timestamp.isoformat(),
+            "fingerprint": fingerprint,
+            "tenant_id": tenant_id,
+        })
```

Comment:
wouldn't it take lot of time to update all the table one by one? 

maybe smth like?
```
UPDATE lastalert
SET first_timestamp = (
    SELECT MIN(alert.timestamp)
    FROM alert
    WHERE alert.fingerprint = lastalert.fingerprint 
    AND alert.tenant_id = lastalert.tenant_id
)
```